packages/config-ui/src/two-choice.jsx

```diff
@@ -29,12 +32,12 @@ class RawNChoice extends React.Component {
 
   render() {
 
-    const { header, className, classes, opts, value } = this.props;
+    const { header, className, classes, opts, value, showVertical } = this.props;
```

Comment:
`showVertical` is missing in PropTypes, also I'm not crazy on `showVertical` as a name. how about `direction: PropTypes.oneOf('horizontal', 'vertical')`, with a default prop of `horizontal`.